### PR TITLE
Vxlan rules are updated

### PR DESCRIPTION
**Pull request by gowrisankarvs**

### CODE REVIEW

File: juniper_official/Solutions/EVPN-VXLAN/irb-next-hops.rule [hunk 1]
```diff
@@ -125,11 +125,6 @@ healthbot {
                     }
                 }
             }
-            variable in-threshold {
```
Comment (gvarao) [inline juniper_official/Solutions/EVPN-VXLAN/irb-next-hops.rule]: Replace min-rate-of-increase function as it is EOL

File: juniper_official/Solutions/EVPN-VXLAN/vtep-interface-count.rule [hunk 2]
```diff
@@ -11,10 +11,67 @@ healthbot {
...
+                helper-files other {
+                    list-of-files VtepCountTable.yml;
+                }
+            }
+        }
         rule check-vtep-interface-count {
             synopsis "VTEP interface count analyzer";
             description "Collects VTEP interfaces count periodically and notifies anomaly when count exceed threshold";
-            rule-frequency 30s;
+            rule-frequency 60s;
```
Comment (gvarao) [inline juniper_official/Solutions/EVPN-VXLAN/vtep-interface-count.rule]: Increase rule frequency from 60 to 65s as refrred rule also has same freq.

File: juniper_official/Solutions/EVPN-VXLAN/vtep-interface-count.rule [hunk 3]
```diff
@@ -29,133 +86,63 @@ healthbot {
...
-                frequency 30s;
-                term Term_1 {
-                    then {
-                        status {
-                            color green;
-                            message "Interface : $vtep-name Endpoint Type: $endpoint-type Endpoint Address: $endpoint-address";
-                        }
-                    }
-                }
+                time-range 60s;
```
Comment (gvarao) [inline juniper_official/Solutions/EVPN-VXLAN/vtep-interface-count.rule]: Increase rule time-range from 60 to 65s as refrred rule also has same freq.

File: juniper_official/Solutions/EVPN-VXLAN/vtep-interface-status.rule [hunk 4]
```diff
@@ -29,14 +37,17 @@ healthbot {
...
-                        matches-with "$vtep-admin-status" up;
+                        matches-with "$admin-status" up {
+                            ignore-case;
+                        }
                     }
                     then {
                         status {
                             color green;
-                            message "VTEP interface admin status is \"$vtep-admin-status\"";
+                            message "VTEP interface admin status is \"$admin-status\"";
```
Comment (gvarao) [inline juniper_official/Solutions/EVPN-VXLAN/vtep-interface-status.rule]: It is good to display($vtep-name) in all term messages

File: juniper_official/Solutions/EVPN-VXLAN/vtep-tunnel-statistics.rule [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+                    then {
+                        status {
+                            color green;
+                            message "$vtep-tunnel output traffic is normal";
+                        }
+                    }
+                }
+            }
+            variable high-threshold-variable {
+                value 80000000000;
```
Comment (gvarao) [inline juniper_official/Solutions/EVPN-VXLAN/vtep-tunnel-statistics.rule]: s/80000000000/800000000/g

File: juniper_official/Solutions/EVPN-VXLAN/vxlan-vport.rule [hunk 6]
```diff
@@ -5,96 +5,97 @@ healthbot {
...
-                when {
-                    greater-than "$vport-percentage-used" 80;
+                term above-threshold {
+                    when {
+                        greater-than "$vport-percentage-used" 80;
+                    }
+                    then {
+                        status {
+                            color red;
+                            message " vport count is $vport-count which is  $vport-percentage-used % of max";
```
Comment (gvarao) [inline juniper_official/Solutions/EVPN-VXLAN/vxlan-vport.rule]: Remove space in begin of message
s/" vport count is $vport-count which is $vport-percentage-used %of max";/"Vport count($vport-count) and used percentage($vport-percentage-used %) is abnormal";/g

File: juniper_official/Solutions/EVPN-VXLAN/vxlan-vport.rule [hunk 7]
```diff
@@ -5,96 +5,97 @@ healthbot {
...
                 }
-                then {
-                    status {
-                        color red;
-                        message " vport count is $vport-count which is  $vport-percentage-used % of max";
+                term inbetween-threshold {
+                    then {
+                        status {
+                            color yellow;
+                            message " vport count is $vport-count which is  $vport-percentage-used  %of max";
```
Comment (gvarao) [inline juniper_official/Solutions/EVPN-VXLAN/vxlan-vport.rule]: Remove space in begin of message

File: juniper_official/Solutions/EVPN-VXLAN/vxlan-vport.rule [hunk 8]
```diff
@@ -5,96 +5,97 @@ healthbot {
...
-                        message " vport count is $vport-count which is  $vport-percentage-used  %of max";
+                frequency 90s;
+                term below-threshold {
+                    when {
+                        less-than "$vport-percentage-used" 60;
+                    }
+                    then {
+                        status {
+                            color green;
+                            message " vport count is $vport-count which is  $vport-percentage-used  %of max";
```
Comment (gvarao) [inline juniper_official/Solutions/EVPN-VXLAN/vxlan-vport.rule]: 1. Remove space in begin of message
2. s/" vport count is $vport-count which is  $vport-percentage-used  %of max";/"Vport count($vport-count) and used percentage($vport-percentage-used  %) is normal";/g

File: juniper_official/Solutions/EVPN-VXLAN/vxlan-vport.rule (hunk 7 above)
Comment (gvarao) [inline juniper_official/Solutions/EVPN-VXLAN/vxlan-vport.rule]: Remove space in begin of message
s/" vport count is $vport-count which is $vport-percentage-used %of max";/"Vport count($vport-count) and used percentage($vport-percentage-used %) is medium";/g

File: juniper_official/Solutions/EVPN-VXLAN/vtep-interface-status.rule [hunk 9]
```diff
@@ -15,9 +16,16 @@ healthbot {
...
+                    frequency 60s;
                 }
             }
+            field vtep-name {
+                sensor vxlan {
+                    path name;
+                }
+                type string;
+                description "Vtep interface name";
+            }
```
Comment (gvarao) [inline juniper_official/Solutions/EVPN-VXLAN/vtep-interface-status.rule]: Is below correct?
"where "oper-status =~ /+*/";"

File: juniper_official/Interfaces/interface-in-errors-iagent.rule [hunk 10]
```diff
@@ -84,12 +82,13 @@ healthbot {
             }
             variable error-threshold-variable {
                 value 1;
-                description "Input and output interface error threshold value";
+                description "Input and output interface error threshold value.";
```
[the remark this answers is not in the document]
Comment (gowrisankarvs) [reply juniper_official/Interfaces/interface-in-errors-iagent.rule]: Updated this rule

File: juniper_official/Protocols/Bgp/bgp-neighbor-flap-iagent.rule [hunk 11]
```diff
@@ -14,13 +14,14 @@
...
  *      yellow, otherwise color is set to green.
+ *
  */
 healthbot {
     topic protocol.bgp {
         description "Monitors BGP neighbour flap count and notify anomalies";
         synopsis "BGP statistics analyzer";
         rule check-bgp-neighbour-flap-iagent {
-            keys neighbor-address;
+            keys peer-address;
```
[the remark this answers is not in the document]
Comment (gowrisankarvs) [reply juniper_official/Protocols/Bgp/bgp-neighbor-flap-iagent.rule]: updated the file

File: juniper_official/Protocols/Bgp/bgp-neighbor-state-iagent.rule [hunk 12]
```diff
@@ -38,7 +38,7 @@ healthbot {
                 frequency 90s;
                 term is-neighbor-up {
                     when {
-                        matches-with "$peer-state" ESTABLISHED;
+                        matches-with "$peer-state" Established;
```
[the remark this answers is not in the document]
Comment (gowrisankarvs) [reply juniper_official/Protocols/Bgp/bgp-neighbor-state-iagent.rule]: updated the file

File: juniper_official/Solutions/EVPN-VXLAN/evpn-mac-count.rule [hunk 13]
```diff
@@ -53,7 +53,7 @@ healthbot {
             trigger mac-count {
                 synopsis "MAC count kpi";
                 description "Sets health based on MAC count in ethernet switching table.";
-                frequency 30s;
+                frequency 90s;
```
[the remark this answers is not in the document]
Comment (gowrisankarvs) [reply juniper_official/Solutions/EVPN-VXLAN/evpn-mac-count.rule]: updated the file

File: juniper_official/Solutions/EVPN-VXLAN/evpn-received-prefix.rule [hunk 14]
```diff
@@ -71,36 +71,34 @@ healthbot {
...
+                frequency 90s;
                 term above-static-threshold {
                     when {
                         greater-than-or-equal-to "$received-count" "$route-count-threshold";
                     }
                     then {
                         status {
                             color red;
-                            message "anomaly in received count!!($received-count)";
+                            message "EVPN received route count($received-count) exceeds static threshold";
```
[the remark this answers is not in the document]
Comment (gowrisankarvs) [reply juniper_official/Solutions/EVPN-VXLAN/evpn-received-prefix.rule]: updated the file

File: juniper_official/Solutions/EVPN-VXLAN/vxlan-vport.rule (hunk 6 above)
Comment (gowrisankarvs) [reply juniper_official/Solutions/EVPN-VXLAN/vxlan-vport.rule]: updated this file

File: juniper_official/Solutions/EVPN-VXLAN/vxlan-vport.rule (hunk 7 above)
Comment (gowrisankarvs) [reply juniper_official/Solutions/EVPN-VXLAN/vxlan-vport.rule]: updated this rule

File: juniper_official/Solutions/EVPN-VXLAN/vtep-interface-status.rule (hunk 9 above)
Comment (gowrisankarvs) [reply juniper_official/Solutions/EVPN-VXLAN/vtep-interface-status.rule]: modified the regex...

File: juniper_official/Solutions/EVPN-VXLAN/evpn-received-prefix.rule [hunk 15]
```diff
@@ -71,36 +71,34 @@ healthbot {
...
-                        equal-to "$received-count-dynamic-threshold" 1 {
-                            time-range 5m;
-                        }
+                        equal-to "$received-count-dynamic-threshold" 1;
                     }
                     then {
                         status {
                             color yellow;
-                            message "anomaly in received count!!($received-count)";
+                            message "EVPN received route count($received-count) exceeds dynamic threshold";
```
[the remark this answers is not in the document]
Comment (gowrisankarvs) [reply juniper_official/Solutions/EVPN-VXLAN/evpn-received-prefix.rule]: updated the rule

File: juniper_official/Solutions/EVPN-VXLAN/vtep-interface-count.rule (hunk 3 above)
Comment (gowrisankarvs) [reply juniper_official/Solutions/EVPN-VXLAN/vtep-interface-count.rule]: updated the rule

**Pull request merged**